week-2-practise/level-1/3-sort.js

```diff
@@ -2,7 +2,14 @@
 // TIP: use an array method
 
 var letters = ["a", "n", "c", "e", "z", "f"];
-var sortedLetters; // ONLY EDIT THIS LINE
+var sortedLetters = [
+  letters[0],
```

Comment:
I don't think this is quite right. If we changed the `letters` array to have different letters or have a different order, then wouldn't log out the correct result.

Try looking up the `.sort()` method on an array (Hint try searching "mdn array sort" on Google)